File: fantasyfb-dotnetcoremvc/Models/Player.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace FantasyFb.Models
{
    public class Player
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Number { get; set; }
        public string Team { get; set; }
        public int QB { get; set; }
        public int RB { get; set; }
        public int WR { get; set; }
        public int TE { get; set; }
        public int K { get; set; }
        [Display(Name = "DST")]
        public int D_ST { get; set; }
        
    }
}

File: fantasyfb-react/src/components/player/Player.js
import React from 'react'

export default function Player( {id, name, number, team, qb, rb, wr, te, k, d_ST }) {

    return (
        <tr>
            <td>{id}</td>
            <td>{name}</td>
            <td>{number}</td>
            <td>{team}</td>
            <td>{qb}</td>
            <td>{rb}</td>
            <td>{wr}</td>
            <td>{te}</td>
            <td>{k}</td>
            <td>{d_ST}</td>
        </tr>
    )
}